src/style/index.css

```diff
@@ -47,7 +47,11 @@ a {
 }
 
 h1,
-h2 {
+h2,
+h3,
+h4,
+h5,
+h6 {
```

Comment:
IMO, looks better applying this to all headings. I think the deepest nesting we have is our ["Aliasing in X" instructions](https://preactjs.com/guide/v10/getting-started#aliasing-in-webpack) and the `<h4>` titles really don't stand out too well. This helps.